mobile/examples/question_answering/android/gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,6 @@
+#Thu Dec 29 10:51:23 PST 2022
+distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```

could we also include the checksum?
https://docs.gradle.org/current/userguide/gradle_wrapper.html#configuring_checksum_verification
```suggestion
distributionSha256Sum=8cc27038d5dbd815759851ba53e70cf62e481b87494cc97cfd97982ada5ba634
distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```